specification/compute/resource-manager/Microsoft.Compute/stable/2021-03-01/compute.json

```diff
@@ -3491,29 +3497,510 @@
+          {
+            "name": "restorePointCollectionName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Restore Point Collection."
+          },
+          {
+            "name": "restorePointName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the restore point."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "202": {
+            "description": "Accepted"
+          },
+          "204": {
+            "description": "No Content"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
+      },
+      "get": {
+        "operationId": "RestorePoints_Get",
+        "description": "The operation to get the restore point.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
           },
           {
-            "$ref": "#/parameters/SubscriptionIdParameter"
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
           },
           {
-            "name": "statusOnly",
-            "in": "query",
-            "required": false,
+            "name": "restorePointCollectionName",
+            "in": "path",
+            "required": true,
             "type": "string",
-            "description": "statusOnly=true enables fetching run time status of all Virtual Machines in the subscription."
+            "description": "The name of the restore point collection."
+          },
+          {
+            "name": "restorePointName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the restore point."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
           }
         ],
+        "x-ms-examples": {
+          "Get a restore point": {
+            "$ref": "./examples/GetRestorePoint.json"
+          }
+        },
         "responses": {
           "200": {
             "description": "OK",
             "schema": {
-              "$ref": "#/definitions/VirtualMachineListResult"
+              "$ref": "#/definitions/RestorePoint"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
             }
           }
-        },
-        "x-ms-pageable": {
```

Comment:
why are we removing the pagination support for the Get Restorepoints api. Dont we need the next link for that

Reply:
The operation to retrieve all restore points in a restore point collection does not return ODataEntityCollection<RestorePoint>, so CRP cannot have pagination support for that.

(I added example response in GetRestorePointCollectionWithContainedRestorePoints)

Instead, the operation only returns a list of restorePoints inside the properties bag of the response.